src/MLJInterface.jl

```diff
@@ -219,6 +238,11 @@ MLJModelInterface.fit(model::MLJInterface.MODELS, verbosity::Int, X, y) = MLJInt
 MLJModelInterface.fit(model::MLJInterface.MODELS, verbosity::Int, X, y, w::Nothing) = MLJInterface.fit(model, verbosity, X, y)
 MLJModelInterface.fit(model::MLJInterface.MODELS, verbosity::Int, X, y, w) = MLJInterface.fit(model, verbosity, X, y, w)
 
+# Add `update` method
+MLJModelInterface.update(model::MLJInterface.MODELS, verbosity::Int, fitresult, cache, X, y) = MLJInterface.update(model, verbosity, fitresult, cache, X, y)
+MLJModelInterface.update(model::MLJInterface.MODELS, verbosity::Int, fitresult, cache, X, y, w::Nothing) = MLJInterface.update(model, verbosity, fitresult, cache, X, y)
```

Comment:
On the left-hand side: Rather than `y, w::Nothing)`, this should be `y, w=nothing)`:

https://alan-turing-institute.github.io/MLJ.jl/dev/adding_models_for_general_use/#Summary-of-methods-1

I think otherwise dispatch will fallback to doing a fit instead of update when user provided weights, I reckon.

Reply:
What this is doing is catching the instance where MLJ defaulting has provided nothing, and on right hand side you can see it just calls function without weights parameter

Line below is where we catch what MLJ passes to us (that isn't `nothing` and we pass it through. I've added tests though.

Reply:
25ee0d4

Reply:
Yes, you're right. I see there is a line above which covers the other case.